src/js/dropdown-menu/dropdown-menu-view.js

```diff
@@ -130,9 +136,9 @@ define([
             },
 
             escKeypressHandler: function(event) {
-                var keyCode = event.keyCode;
+                var key = event.keyCode;
 
-                if (keyCode === constants.keyCodes.esc) {
+                if (_.contains(this.keyClose, key)) {
                     // When the ESC key is pressed, close all menus
                     this.closeDropdownMenus(true);
```

Comment:
Why is this plural? Won't each dropdown have only a single menu? Is it closing all menus on the page? I'm not a big fan of affecting other views from within this view. Pressing escape won't close other modals, so why would it understand other dropdowns?